core/templates/services/audio-player.service.ts

```diff
@@ -52,9 +52,6 @@ export class AudioPlayerService {
   ) {}
 
   private async _loadAsync(filename: string, successCallback, errorCallback) {
-    if (this._loadingTrack) {
-      throw new Error('Already loading a track... Please try again!');
-    }
```

Comment:
Why is this code remove, do we know why this was added in the first place? (not sure whether it handles any unexpected situation?)

Reply:
`this._loadingTrack` will remain false and does not change it's value

Reply:
The variable value is set as true when the service is initialized.
https://github.com/oppia/oppia/blob/a934619517eee3fc7ec787911a7afe2798b14cd5/core/templates/services/audio-player.service.ts#L42
but is never set to true anywhere else in the code.

Reply:
Hm, if `this._loadingTrack` is not really used, should we remove all occurrences of it?

Reply:
Removed